modules/apps/journal/journal-web/src/main/resources/META-INF/resources/js/export_translation/ExportTranslationModal.es.js

```diff
@@ -191,12 +191,6 @@ const ExportTranslationModal = ({
 							/>
 						))}
 					</ClayForm.Group>
-
-					<ClayInput
-						name={`_${namespace}_articleIdsIds`}
-						type="hidden"
-						value={articleIds}
-					/>
```

Comment:
Is it working without this input?

Because I'm not seeing where pass values in the form in other input or something.

Reply:
Yeah, tested and working... as I commented in the commit, we are not passing the form values, but sending the params via url [here](https://github.com/liferay-lima/liferay-portal/blob/1dccf5195e0108dcdb9ec321731414f3be12672b/modules/apps/journal/journal-web/src/main/resources/META-INF/resources/js/export_translation/ExportTranslationModal.es.js#L53) and [here](https://github.com/liferay-lima/liferay-portal/blob/1dccf5195e0108dcdb9ec321731414f3be12672b/modules/apps/journal/journal-web/src/main/resources/META-INF/resources/js/export_translation/ExportTranslationModal.es.js#L161).